doc/api/readline.md

```diff
@@ -527,6 +528,107 @@ rl.on('line', (line) => {
 });
 ```
 
+## TTY keybindings
+
+<table>
+  <tr>
+    <th>Keybindings</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td><code>ctrl+shift+backspace</code></td>
+    <td>delete line left</td>
+  </tr>
+  <tr>
+    <td><code>ctrl+shift+delete</code></td>
+    <td>delete line right</td>
+  </tr>
+  <tr>
+    <td><code>ctrl+c</code></td>
+    <td>emits SIGINT</td>
+  </tr>
+  <tr>
+    <td><code>ctrl+h</code></td>
+    <td>delete left</td>
+  </tr>
+  <tr>
+    <td><code>ctrl+d</code></td>
+    <td>delete right or EOF</td>
+  </tr>
+  <tr>
+    <td><code>ctrl+u</code></td>
+    <td>delete from current to line start</td>
+  </tr>
+  <tr>
+    <td><code>ctrl+k</code></td>
+    <td>delete from current to end of line</td>
+  </tr>
+  <tr>
+    <td><code>ctrl+a</code></td>
+    <td>goto start of line</td>
+  </tr>
+  <tr>
+    <td><code>ctrl+e</code></td>
+    <td>goto to end of line</td>
+  </tr>
+  <tr>
+    <td><code>ctrl+b</code></td>
+    <td>back one character</td>
+  </tr>
+  <tr>
+    <td><code>ctrl+f</code></td>
+    <td>forward one character</td>
+  </tr>
+  <tr>
+    <td><code>ctrl+l</code></td>
+    <td>clear screen </td>
+  </tr>
+  <tr>
+    <td><code>ctrl+n</code></td>
+    <td>next history item</td>
+  </tr>
+  <tr>
+    <td><code>ctrl+p</code></td>
+    <td>prev history item </td>
```

Comment:
`prev` -> `previous` for more formal style?

Reply:
done.